lib/formats/deps.js/parser.js

```diff
@@ -29,6 +29,11 @@ module.exports = function parse(depsData) {
         const data = [].concat(record.data);
 
         data.forEach(dep => {
+            if (!dep) {
+                console.warn(`Empty deps.js block in ${record.path}. Ignore it.`);
```

Comment:
At first look, you are testing every object came in deps. For example, if we got smth like this:
```javascript
shouldDeps: [ false, false ]
```
We will get two warn messages, isn't it?